util/make_timeline.py

```diff
@@ -284,8 +375,9 @@ def main(args):
     parser.add_argument('-rf', '--fight', type=int, help="Fight ID of the report to use. Defaults to longest in the report")
 
     # Log file arguments
-    parser.add_argument('-s', '--start', type=timestamp_type, help="Timestamp of the start, e.g. '12:34:56.789")
-    parser.add_argument('-e', '--end', type=timestamp_type, help="Timestamp of the end, e.g. '12:34:56.789")
+    parser.add_argument('-s', '--start', help="Timestamp of the start, e.g. '12:34:56.789")
+    parser.add_argument('-e', '--end', help="Timestamp of the end, e.g. '12:34:56.789")
+    parser.add_argument('-lf', '--search_fights', nargs=1, type=int, help="Encounter whose start/stop times should be used, e.g. '1'. -1 to list all.")
```

Comment:
I can't remember how parseargs work, but is it possible to have `-lf` on its own (with no magic -1 value) do the automatic listing?

Reply:
From what I can see, it should be possible with `nargs='?', type=int, const=-1`. The brief change I made to permit that seems to work okay, but I'll want to test it more before turning it in.

Reply:
Well, that wasn't too difficult. I'm not totally happy with the implementation, but it works and overall just makes more sense to the user.